### PR TITLE
Fix for issue #117

### PR DESCRIPTION
**Pull request by antjori**
Fixes issue #117 related to influxdb/influxdb/issues/2420.

### CODE REVIEW

File: src/main/java/org/influxdb/dto/Point.java [hunk 1]
```diff
@@ -286,22 +286,26 @@ private StringBuilder concatenateFields() {
...
-			} else if (value instanceof Number) {
-				sb.append(numberFormat.format(value));
-			} else {
-				sb.append(value);
-			}
 
-			if (loops < fieldCount) {
-				sb.append(",");
+			// Avoiding invalid null type
+			if (value != null) {
```
Comment (majst01) [inline src/main/java/org/influxdb/dto/Point.java]: I think it is more readable if you continue if value == null, it also leads to less idented code. 
If you mind change it this way i will merge

**Comment by antjori** [comment]
Sure, I'll do it!

File: src/main/java/org/influxdb/dto/Point.java (hunk 1 above)
Comment (antjori) [inline src/main/java/org/influxdb/dto/Point.java]: Sure, I'll do it!

**Comment by antjori** [comment]
Since I didn't create a branch for this issue, I'm going to create it and do all the changes there.

Afterwards, I'll do a new pull request.

Closing this one.

**Pull request closed without merging**